docs/index.rst

```diff
@@ -36,7 +36,7 @@ Or you can install from source:
 
    git clone https://github.com/profjsb/deepCR.git
    cd deepCR/
-   pip install
+   python setup.py install
```

Comment:
pip install is preferred. Please revert. https://stackoverflow.com/questions/15724093/difference-between-python-setup-py-install-and-pip-install